polychromatic-controller

```diff
@@ -1478,6 +1478,7 @@ class Controller(object):
                     ["lighting_pulsate", "pulsate", _("Pulsate")],
                     ["lighting_ripple", "ripple", _("Ripple")],
                     ["lighting_starlight_random", "starlight", _("Starlight")],
+                    ["lighting_reactive", "system_monitor", _("System Monitor")],
```

Comment:
I thought that this was a good guess for what might be a keyboard, but maybe you can suggest something better?

Reply:
This could include mice and the mouse mat too, although there's no reason why they couldn't work if they support a matrix (with more then one row)